internal/cmd/config/config.go

```diff
@@ -387,6 +397,24 @@ func Parse(d string) (*Config, error) {
 			result.Controller.GracefulShutdownWaitDuration = t
 		}
 
+		if result.Controller.Scheduler != nil {
+			if result.Controller.Scheduler.JobRunInterval != "" {
```

Comment:
does the hcl parser guarantee that the interface will not be a nil value, and default it to an empty string?

Reply:
It seems like it must since other examples do this and the test with a scheduler block and no intervals does parse correctly.